src/docs/components/methods.md

```diff
@@ -51,9 +51,9 @@ Stencil's architecture is async at all levels which allows for many performance
 
 
 ```tsx
-// VALID: using async
+// VALID: using "async" keyword
 @Method()
-async myMethod() {
+async myMethod() /* returnType: Promise<number> */ {
```

Comment:
It might cause confusion to mention `returnType` some new developers might thing it's a new thing, i would rather explain in the comment above that a async function/method implicitally returns a Promise, so the return type is `Promise<number>`